src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -586,6 +613,24 @@ void Increment(ref long counter)
         GC.KeepAlive(listener);
     }
 
+    private static (string, string) GetGetQueryParameters(string contentSource, ClientContext clientContext)
+    {
+        StringBuilder queryString = new StringBuilder();
+        queryString.Append($"?Var{0}={contentSource}");
+        string expectedString = contentSource;
+
+        int num = clientContext.GetRandomInt(100);
+
+        for (int i = 1; i < num; i++)
+        {
+            string vari = clientContext.GetRandomSubstring(contentSource);
```

Comment:
My only concern here is that the query string can end up being _very_ large, bounded by `100 * maxContentLength`. Should maybe be less than `maxContentLength` or some other size parameter?

Reply:
Maybe renaming `maxContentLength` to something nondescript like `maxSize`?